bigbang/domain.py

```diff
@@ -3,14 +3,54 @@
 """
 
 import math
-import re
-
 import pandas as pd
+import re
 
 email_regex = r"[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+"
 domain_regex = r"[@]([a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)$"
 
 
+def domain_entropy(domain, froms):
```

Comment:
This method might be outside of the scope which this file is supposed to have "Functions for extracting parts of an email address, such as its domain.". As this is method is used post-scraping, to analyse the mailing archive/list I would place this method in the /bigbang/analysis/ subfolder. This folder will need restructuring soon too as the number of methods is growing there making the current setup unwieldy.
What would you think about placing this method simple in /bigbang/analysis/domain.py for now?